src/main/java/it/gov/pagopa/pu/organization/repository/BrokerRepository.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.organization.repository;
+
+import it.gov.pagopa.pu.organization.entity.BrokerEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "broker", path = "broker")
```

Comment:
let's use the plural for REST paths on entities
```suggestion
@RepositoryRestResource(collectionResourceRel = "broker", path = "brokers")
```

Reply:
this has not been resolved